### DISCUSSION TITLE
https://ga-dev-tools.web.app/hit-builder/ no me deja usar la herramienta

### DISCUSSION THREAD

**Discussion by nvallss** [community/community#46974] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hola

La herramienta para eliminar datos no validos en analytics no me funciona

https://ga-dev-tools.web.app/hit-builder/

Mensaje que me sale: 
Hit Builde
![ga](https://user-images.githubusercontent.com/124363607/218102557-7c74ef60-7855-4681-9f71-f0dde276ceba.jpg)
r
Login status could not be determined. Please ensure cookies are enabled.

**Comment by queenofcorgis** [comment]
Hi @nvallss ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this site. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
